app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -318,6 +321,8 @@ class DeadlineDetailsActivity : AppCompatActivity() {
             dateView.text =
                 getString(R.string.DueTheXatX, dateTime.toLocalDate(), dateTime.toLocalTime())
             titleView.text = SpannableStringBuilder(title)
+            if (deadline.locationName != null) locationView.text = deadline.locationName
+            locationView.text = deadline.locationName
```

Comment:
I think there is a problem here...

Reply:
Oh yes your're right. Thanks. About Mapbox, I created a new activity called SearchLocation that contains the mapbox view. This activity is lauched using the button on the addDeadlineActivity and the location result is communicated via intent. This is done using the StartActivityForResult() function.

Reply:
So it this "solution" give us the ability to execute addDeadlineTests without any mapbox stuff in it.

Reply:
OK nice. I am just wondering where the coverage drop comes from... From CodeClimate I see that `SearchLocationActivity` has 0% coverage, is that correct?